src/System.Private.Reflection.Metadata.Ecma335/src/System.Private.Reflection.Metadata.Ecma335.csproj

```diff
@@ -0,0 +1,262 @@
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Metadata\Signatures\SignatureHeader.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Metadata\Signatures\SignatureKind.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Metadata\Signatures\SignatureTypeCode.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Metadata\TypeSystem\StandaloneSignature.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Metadata\TypeSystem\TypeDefinition.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Metadata\TypeSystem\TypeLayout.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Metadata\TypeSystem\TypeReference.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Metadata\TypeSystem\TypeSpecification.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\CoffHeader.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\CorFlags.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\CorHeader.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\DebugDirectory\CodeViewDebugDirectoryData.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\DebugDirectory\DebugDirectoryEntry.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\DebugDirectory\DebugDirectoryEntryType.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\DirectoryEntry.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\Machine.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\PEBinaryReader.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\PEFileFlags.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\PEHeader.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\PEHeaders.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\PEMemoryBlock.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\PEReader.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\PEStreamOptions.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\PortableExecutable\SectionHeader.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\Throw.cs" />
+    <Compile Include="$(SystemReflectionMetadataPath)System\Reflection\System.Reflection.cs" />
```

Comment:
Wait, it looks like the file listings are just duplicated. In that case, could we just revert the $(SystemReflectionMetadataPath) in original csproj?